Dockerfile

```diff
@@ -0,0 +1,41 @@
+# Dockerfile for cBioPortal
+FROM elementolab/jamm:1.0
```

Comment:
I have not tested everything, but I believe these are the Debian packages that contain all the dependencies, including runtime dependencies of the database upgrade script and dataset validator:

```Dockerfile
RUN apt-get update && apt-get install -y --no-install-recommends \
                git \
                maven \
                libmysql-java \
                openjdk-8-jdk="$JAVA_DEBIAN_VERSION" \
                python \
                python-jinja2 \
                python-mysqldb \
                python-requests \
        && rm -rf /var/lib/apt/lists/*
# set up Tomcat to use the MySQL Connector/J Java connector
RUN ln -s /usr/share/java/mysql-connector-java.jar "$CATALINA_HOME"/lib/
```